media/img/contribute/icon/cal.svg

```diff
@@ -1,7 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
```

Comment:
This file (and the other three icon SVGs) is still in use on the [`media/css/mozorg/contribute.scss`](https://github.com/alexgibson/bedrock/blob/contribute-lang/media/css/mozorg/contribute.scss#L379-L393) file. These SVG icons being removed are showing as 404'ing in the console: 

<img width="631" alt="image" src="https://user-images.githubusercontent.com/2692333/108766346-a7d0a380-751a-11eb-90f1-f02bdc308626.png">